### ISSUE TITLE
The module is completely broken for me

### ISSUE THREAD

**Issue by William04A** [gfleetwood/tts-watson#3]
The first issue I got while installing the package was this:

```
print "Transform '" + str(text) + "' into sound"
                      ^
SyntaxError: invalid syntax

```
I fixed the syntax of the print statements, but then I got the following error:
```
dataToRead += data
TypeError: must be str, not bytes
```

I am using Python 3.6 and I have all the requirements mentioned installed. My code is:
```
from tts_watson.TtsWatson import TtsWatson
tts = TtsWatson("**Username**", "**Password**", "en-US_Allisonvoice")
tts.play("Test")
```
I also got a InsecureRequestWarning.

**Comment by iMaKaay** [comment]
Were you able to solve the problem? i have the same problem

**Comment by William04A** [comment]
@iMaKaay No. I quickly realized that this module most likely is outdated and doesn´t work with Python 3.

**Comment by gubgbugbugvjgvugv** [comment]
I had the same problem I solved the errors by converting the .py files with 2to3py.Now I get no errors and no output too :(

**Comment by William04A** [comment]
@gubgbugbugvjgvugv that´s a bummer. Are you sure that you converted the whole library and not just a single file?
